bindings/protocols/coap/index.html

```diff
@@ -1,181 +1,464 @@
+                </tr>
+            </tbody>
+        </table>
+    </section>
+    <section id="url">
+        <h2>URI Schemes</h2>
+        <p>
+            CoAP uses the following URI schemes for identifying CoAP resources
+            and providing a means of locating the resource:
+        </p>
+        <table class="def">
+            <thead>
+                <tr>
+                    <th>URI Scheme</th>
+                    <th>Reference</th>
+                </tr>
+            </thead>
+            <tbody>
+                <tr>
+                    <td>coap</td>
+                    <td>[[RFC7252]]</td>
+                </tr>
+                <tr>
+                    <td>coap+tcp</td>
+                    <td>[[RFC8323]]</td>
+                </tr>
+                <tr>
+                    <td>coap+ws</td>
+                    <td>[[RFC8323]]</td>
+                </tr>
+                <tr>
+                    <td>coaps</td>
+                    <td>[[RFC7252]]</td>
+                </tr>
+                <tr>
+                    <td>coaps+tcp</td>
+                    <td>[[RFC8323]]</td>
+                </tr>
+                <tr>
+                    <td>coaps+ws</td>
+                    <td>[[RFC8323]]</td>
+                </tr>
+            </tbody>
+        </table>
+        <p>
+            See <a href="https://www.iana.org/assignments/uri-schemes/">Uniform Resource Identifier (URI) Schemes</a>
+            for a complete list of CoAP URI schemes.
+        </p>
+    </section>
+    <section id="vocabulary">
+        <h2>CoAP Vocabulary</h2>
+        <p>
+            This section describes the vocabulary used in CoAP. A protocol
+            binding implementation should use the vocabulary defined in this
+            section to describe the different configuration that can be used to
+            exchanged data between Web of Things.
+        </p>
+        <section>
+            <h3>Form terms</h3>
+            <table class="def">
+                <thead>
+                    <tr>
+                        <th>Vocabulary term</th>
+                        <th>Description</th>
+                        <th>Assignment</th>
+                        <th>Type</th>
+                    </tr>
+                </thead>
+                <tbody>
+                    <tr>
+                        <td><code>coap:method</code></td>
+                        <td>Indicates the CoAP method to be used in the request</td>
+                        <td>optional</td>
+                        <td><a href="#Method">Method</a></td>
+                    </tr>
+                    <tr>
+                        <td><code>coap:format</code></td>
+                        <td>Indicates that a payload and a CoAP Content-Format option [[RFC7252]] be included in the request</td>
+                        <td>optional</td>
+                        <td><a href="#ContentFormat">ContentFormat</a> or array of <a href="#ContentFormat">ContentFormat</a></td>
```

Comment:
According to [RFC 7252 section 5.10](https://datatracker.ietf.org/doc/html/rfc7252#section-5.10), the Content-Format option is not repeatable. Is it intentional to allow an array of Content-Formats here?

Reply:
The thought was that a server actually might accept multiple content-formats (e.g., both SenML/CBOR and SenML/JSON), which could be listed in the TD (probably in the order of the server's preference). A client would pick the most appropriate one and include a payload (and Content-Format option) accordingly.

Reply:
Oh, I see. This would also be consistent with the `ct` link parameter from the CoRE RD draft, so I think that sounds good :)